apps/test/unit/templates/manageStudents/ManageStudentsAgeCellTest.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import sinon from 'sinon';
+import {UnconnectedManageStudentAgeCell as ManageStudentAgeCell} from '@cdo/apps/templates/manageStudents/ManageStudentsAgeCell';
+
+
+describe('ManageStudentAgeCell', () => {
+  const DEFAULT_PROPS = {
```

Comment:
nit: these default props are typically defined outside of the describe block, like you had in an earlier version